library/src/main/java/com/chuckerteam/chucker/internal/ui/transaction/TransactionPayloadFragment.kt

```diff
@@ -268,16 +268,16 @@ internal class TransactionPayloadFragment :
             if (type == PayloadType.RESPONSE && responseBitmap != null) {
                 val bitmapLuminance = responseBitmap.calculateLuminance()
                 result.add(TransactionPayloadItem.ImageItem(responseBitmap, bitmapLuminance))
-            } else if (!isBodyPlainText) {
-                requireContext().getString(R.string.chucker_body_omitted).let {
-                    result.add(TransactionPayloadItem.BodyLineItem(SpannableStringBuilder.valueOf(it)))
-                }
-            } else {
-                if (bodyString.isNotBlank()) {
-                    bodyString.lines().forEach {
+            } else if (bodyString.isBlank()) {
+                if (!isBodyPlainText) {
```

Comment:
Can you merge this into a single condition?

Reply:
Not in the current format, but I think it makes more sense to change to `when`. It'll make it clearer.